web/README.md

```diff
@@ -54,6 +55,8 @@ The first place is the browser's console which can give
 some hints. The second location to check for errors or warnings is output of `npm run server`
 where you can find issues when communicating with the backend. And last but on least is
 journal on backend machine where is logged backend activity `journalctl -b`.
+If journal does not provide required info, there is also option to see dbus communication
+which can give hint about data flow. Command is `busctl monitor --address /run/agama/bus`
```

Comment:
I think the address is `unix:path=/run/agama/bus`

Reply:
right